packages/expo-cli/src/commands/upload.js

```diff
@@ -40,6 +41,10 @@ export default program => {
       '--apple-id <apple-id>',
       'your Apple ID username (you can also set EXPO_APPLE_ID env variable)'
     )
+    .option(
+      '--itc-team-id <itc-team-id>',
+      'iTunes connect Team ID (optional if there is only one team available)'
```

Comment:
iTunes Connect has been renamed to App Store Connect.

Reply:
I'm leaving --itc-team-id because this name is also used in interactive fastlane prompt and don't want to confuse users

Reply:
If you have many teams and you didn't pass `--itc-team-id`, what does Fastlane print (in non-interactive mode)? Can you copy-paste the output here?

I think the error message we show needs to tell that you should pass the `--itc-team-id` flag to the command in that case. (I believe the default message in Fastlane might refer to the `FASTLANE_ITC_TEAM_ID` environment variable instead.)

Reply:
```
Ensuring the app exists on App Store Connect, this may take a while...

+----------------+--------------------------------------+
|              Summary for produce 2.112.0              |
+----------------+--------------------------------------+
| app_identifier | com.testExpo.testProject1233iut1ut2  |
|                | fre7r32r                             |
| app_name       | testProject                          |
| username       | turtle-dev@sixfivezero.net           |
| language       | English                              |
| sku            | 1550147107                           |
| platform       | ios                                  |
| skip_itc       | false                                |
| skip_devcenter | false                                |
+----------------+--------------------------------------+

[13:25:09]: [DevCenter] App 'com.testExpo.testProject1233iut1ut2fre7r32r' already exists, nothing to do on the Dev Center
Multiple App Store Connect teams found, please enter the number of the team you want to use: 
Note: to automatically choose the team, provide either the App Store Connect Team ID, or the Team Name in your fastlane/Appfile:
Alternatively you can pass the team name or team ID using the `FASTLANE_ITC_TEAM_ID` or `FASTLANE_ITC_TEAM_NAME` environment variable

  itc_team_id "119590309"

or

  itc_team_name "650 INDUSTRIES INC. - M7Y5F9WYK2"

1) "650 INDUSTRIES INC. - M7Y5F9WYK2" (119590309)
2) "650 Industries, Inc." (17102800)
3) "Alicja Warchał" (118989844)
4) "SOFTWARE MANSION SP Z O O SPOLKA KOMANDYTOWA" (118729549)
```

Reply:
```
[14:16:22] Failed to upload the standalone app to the app store.
[14:16:22] Multiple App Store Connect Teams found; unable to choose, terminal not interactive!
```